Composer/packages/client/src/recoilModel/selectors/dialogImports.ts

```diff
@@ -36,10 +36,11 @@ export const getLanguageFileImports = <T extends LgFile | LuFile | QnAFile>(
       continue;
     }
     const currentImports = file.imports.map((item) => {
+      const importedFile = getFile(getBaseName(item.id));
       return {
         displayName: item.description,
         importPath: item.path,
-        id: getBaseName(item.id),
+        id: importedFile ? importedFile.id : '',
```

Comment:
```importedFile?.id ?? ''```

Reply:
the `id` needs to be string type, otherwise ESlint would complained of it